### ISSUE TITLE
Better distinction or separation between personal note and connection's own bio in web UI

### ISSUE THREAD

**Issue by rubcap** [mastodon/mastodon#32310]
### Pitch

I just discovered personal notes. Very handy! But when I add a personal note, it seems to be part of my connection's own bio. There is no visual distinction between the personal note text and the bio text.

I'd suggest giving the personal note and its title a pale gray background, with enough contrast for accessibility reasons.

### Motivation

Immediately see where one's personal note ends, and the connection's bio starts.
Every user of personal notes would benefit from this.

**Comment by nclm** [comment]
Yes! I came here to report that too :) Since I took it, here’s a screenshot:

![Image](https://github.com/user-attachments/assets/602b2a11-88d9-4b99-a876-36d4473d1793)

And as a proof of concept only (not a design mock-up), an example of how clearer it can be with a separation:

![Image](https://github.com/user-attachments/assets/987b3741-902a-41fa-bf0e-3a6cdc3af4cf)

**Comment by renchap** [comment]
Solved in #37593
